src/components/Editor.jsx

```diff
@@ -73,6 +76,13 @@ class Editor extends React.Component {
     if (openLine) {
       this.emphasizeLine(openLine);
     }
+    if (value && AST.children && filters) {
```

Comment:
This looks like it will override the annotation highlighting that is done by `this.emphasizeLine`. 

I just tested annotation highlighting in this branch and master, and when a line is being annotated/viewed, it is no longer emphasized (another regression). So it won't be possible to know for sure if this will interfere with it or not, as it doesn't actually work anymore. 

However, I think adding a condition that verifies no lines are currently being emphasized will prevent this fix from interfering with the `emphasizeLine` regression fix when it is put in place.